src/usr/local/www/vpn_ipsec_phase2.php

```diff
@@ -427,9 +441,11 @@
 
 $localid_help_tunnel  = "Local network component of this IPsec security association.";
 $localid_help_vti     = "Local point-to-point IPsec interface tunnel network address.";
+$localid_help_bypass  = "Local network that bypasses from IPsec processing for this IPsec connection.";
 $localid_help_mobile  = "Network reachable by mobile IPsec clients.";
 $remoteid_help_tunnel = "Remote network component of this IPsec security association.";
 $remoteid_help_vti    = "Remote point-to-point IPsec interface tunnel network address.";
+$remoteid_help_bypass = "Remote network that bypasses from IPsec processing for this IPsec connection.";
```

Comment:
Remove "from"

Reply:
Removed,
But it’s true - if you add bypass rule, it's bypassed for all peers